app/data-generation/get-delta-amount.js

```diff
@@ -1,19 +1,18 @@
 const db = require('../data')
 const { getValue } = require('./get-value')
+const { getDataFilter } = require('../get-data-filter')
 
 const getDeltaAmount = async (event, transaction) => {
   const requestNumber = event.data.paymentRequestNumber
   const value = getValue(event)
   if (requestNumber === 1 || value === null) {
     return value
   }
+  const where = getDataFilter(event.data)
+  console.log(db) // Check the value of db
```

Comment:
Do we need these console logs? Fine if it makes sense to keep them, but given this will log for every event we receive, it could be beneficial to remove.